tests/Core/EngineTest.php

```diff
@@ -72,14 +72,41 @@ public function testCrawlsStartUrls(): void
             $this->makeRequest('http://localhost:8000/test1'),
             $this->makeRequest('http://localhost:8000/test2'),
         ];
-        $run = new Run($startRequests);
+        $run = new Run($startRequests, 'DefaultSpider');
```

Comment:
If the actual value of a parameter in a test doesn't matter and we only care about what it represents, we use `::this-notation::`. This applies to all other places that pass the new namespace parameter, too.

```suggestion
        $run = new Run($startRequests, '::namespace::');
```

I've written a blog post on this topic in case you're interested in the rationale behind this: https://www.kai-sassnowski.com/post/reducing-unnecessary-details-in-tests/

Reply:
That actually makes a lot of sense, cheers!